app/src/main/java/com/eveningoutpost/dexdrip/UtilityModels/BgSendQueue.java

```diff
@@ -39,6 +40,10 @@
     @Column(name = "operation_type")
     public String operation_type;
 
+    private static Context mContext = Home.getContext();
```

Comment:
Yeah, the problem is I need to create a single PebbleSync. If I put it in the addToQueue function, a new PebbleSync is created every time a Bg is added to the queue. If this is better done elsewhere, let me know. But for now it stays where it is. 
I tried putting in code to only create/set pebbleSync once, if pebbleSync was null, but it always created new ones. Happy to try another method. 
All addToQueue needs to do is pebbleSync.sendData if broadcast_to_pebble is true. 

I hate emailing on my phone on the train:/


 
John Stevens
M - +61 418526349

> On 20 May 2015, at 00:08, Stephen Black <notifications@github.com> wrote:
> 
> In app/src/main/java/com/eveningoutpost/dexdrip/UtilityModels/BgSendQueue.java:
> 
> > @@ -39,6 +40,10 @@
> >      @Column(name = "operation_type")
> >      public String operation_type;
> >  
> > +    private static Context mContext = Home.getContext();
> Each time we enter this class we always come in through the same function and it already has a context passed in, we should not reach out to home to get its context, I changed this in the other branch already (this is what triggers those crashes)
> 
> —
> Reply to this email directly or view it on GitHub.
>